packages/strapi-plugin-email/config/functions/bootstrap.js

```diff
@@ -18,4 +18,25 @@ const createProvider = emailConfig => {
 module.exports = async () => {
   const emailConfig = _.get(strapi.plugins, 'email.config', {});
   strapi.plugins.email.provider = createProvider(emailConfig);
+
+  // Add permissions
```

Comment:
I think we should add the permissions into the settings tab instead. Do you confirm @alexandrebodin ?

To do so:


```
const actions = [
  {
    section: 'settings',
    category: 'email',
    displayName: 'Read',
    uid: 'settings.read',
    subCategory: 'settings',
    pluginName: 'email',
  },
  {
    section: 'settings',
    category: 'email',
    displayName: 'Send test emails',
    uid: 'settings.test',
    subCategory: 'settings',
    pluginName: 'email',
  },
];
```

Reply:
Yes :) Any permission realted to the Settings panel should be put in the settings tab @MattieBelt :)

Reply:
Moved them to settings section